sql/src/main/antlr/OpenDistroSQLParser.g4

```diff
@@ -370,3 +371,11 @@ functionArg
     : expression
     ;
 
+arrayArgs
```

Comment:
I think it should be collection of expression instead of array. (Here are the reference from bigquery)[https://cloud.google.com/bigquery/docs/reference/standard-sql/operators#in_operators].

Reply:
is it a naming change or a structural change?
Would renaming arrayArgs to `expressionCollection` be the only thing to change?

Reply:
This is the current limitation with the Function/Operator definition and resolver which doesn't support VAR paramaters. Most of the implementation is pretty good, my only concern is it is not ARRAY data type. Looking for the solution to support VAR paramaters now.

Reply:
Could you please add some details on next steps?

From my understanding you suggest to discard this PR away and add implementation for VARs.
1) Is this correct?
2) is there any plan/timeline on your side?

Thank you,
Francesco

Reply:
The ARRAY is also make sense, but the defintion of IN operator should be IN(BOOL, ARRAY(BOOL)). which required more change in Type (support ARRAY(E)), FunctionSignature.
Add my thoughts in [here](https://github.com/penghuo/sql/commit/997042319a2fc1e860e9ddd31bbe7e445826ebcc). Does it make sense to you?